website/content/docs/connect/manage-traffic/route-to-local-upstreams.mdx

```diff
@@ -83,23 +83,252 @@ You can configure the default routing behavior for all proxies in the mesh as we
+
+```json
+{
+  "@type": "type.googleapis.com/envoy.config.endpoint.v3.ClusterLoadAssignment",
+  "cluster_name": "web.default.dc1.internal.161d7b5a-bb5f-379c-7d5a-1fc7504f95da.consul",
+  "endpoints": [
+    {
+      "lb_endpoints": [
+        {
+          "endpoint": {
+            "address": {
+              "socket_address": {
+                "address": "10.42.2.6",
+                "port_value": 20000
+              }
+            },
+            "health_check_config": {}
+          },
+          ...
+        },
+        ...
+      ],
+      "locality": {}
+    },
+    {
+      "lb_endpoints": [
+        {
+          "endpoint": {
+            "address": {
+              "socket_address": {
+                "address": "10.42.3.6",
+                "port_value": 20000
+              }
+            },
+            "health_check_config": {}
+          },
+          ...
+        },
+        ...
+      ],
+      "locality": {},
+      "priority": 1
+    },
+    {
+      "lb_endpoints": [
+        {
+          "endpoint": {
+            "address": {
+              "socket_address": {
+                "address": "10.42.0.6",
+                "port_value": 20000
+              }
+            },
+            "health_check_config": {}
+          },
+          ...
+        },
+        ...
+      ],
+      "locality": {},
+      "priority": 2
+    }
+  ],
+  ...
+}
+```
+
+To force a failover for testing purposes, scale the upstream service instances in the downstream's local zone (or region, if no local zone instances are avaiable) to zero.
+
+<Tip>
+Keep in mind the following:
+  - Priority for failover is in ascending order, starting with `0`. Priority `0` (the "highest" priority) will not be explicitly visible in xDS output, since it is the default value for that field.
+  - Once Envoy failover configuration is in place, the specific timing of failover is determined by the downstream Envoy proxy, not Consul. Consul health status may not correspond 1-1 with Envoy's failover behavior, which is also dependent on outlier detection.
+</Tip>
+
+## Troubleshooting
+
+If you do not see the expected priorities, verify that your locality configuration is in place and that you have enabled `PrioritizeByLocality` in your proxy defaults or service resolver. If `PrioritizeByLocality` is enabled but the local proxy lacks locality configuration, Consul will emit a warning log to indicate that the policy could not be applied:
```

Comment:
```suggestion
If you do not see the expected priorities, verify that locality is configured in the Consul agent and that `PrioritizeByLocality` is enabled in your proxy defaults or service resolver configuration entry. When `PrioritizeByLocality` is enabled but the local proxy lacks locality configuration, Consul emits a warning log to indicate that the policy could not be applied:
```
We should copy or move this to the Common Error Messages page: https://developer.hashicorp.com/consul/docs/troubleshoot/common-errors

Reply:
Ah, I hadn't considered that page. I don't think I'd expect this error to be common FWIW, but happy to add it there if it's meant to be a catch-all.

I think it's more likely that a user would notice this error after being told by the feature docs to look for it as a troubleshooting step, rather than seeing it and trying to figure out what to do next, so I'd lean toward at least calling it out here.

Reply:
Fixed per the above suggestion - LMK what you think re: adding to common errors.